src/key.js

```diff
@@ -771,6 +771,84 @@ Key.prototype.verifyAllUsers = async function(keys) {
   return results;
 };
 
+/**
+ * generate a new subkey to a Key
```

Comment:
Should be "Adds a new subkey to a key"

Reply:
"generate and add a new subkey to a Key"